server/public/model/scheduled_post.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+import "net/http"
+
+type ScheduledPost struct {
+	Draft
+	Id          string `json:"id"`
+	ScheduledAt int64  `json:"scheduled_at"`
+	ProcessedAt int64  `json:"processed_at"`
```

Comment:
It was recommended by Chris P for scheduled job. I'll store the timestamp when the scheduled post for processed and it wasn't posted successfully. This will help us track delays in posting scheduled posts and probably help in the new scheduled job changes. Check out the technical specs at https://mattermost.atlassian.net/wiki/spaces/ICU/pages/2971500547/Tech+Spec+Scheduled+Messages